### PR TITLE
Update i18n.tsx

### PR DESCRIPTION
**Pull request by leandro-costa**
Update i18n.tsx to add all translations of i18n folder

### CODE REVIEW

File: webapp/src/i18n.tsx [hunk 1]
```diff
@@ -1,44 +1,62 @@
...
+        return messages_sv
     case 'tr':
         return messages_tr
     case 'zh-cn':
         return messages_zhHant
     case 'zh-tx':
         return messages_zhHans
     case 'zh-tw':
-        return messages_zhHans
+        return messages_zhHans
```
Comment (jespino) [inline webapp/src/i18n.tsx:59]:
```suggestion
        return messages_zhHans
```

File: webapp/src/i18n.tsx [hunk 2]
```diff
@@ -1,44 +1,62 @@
...
+import messages_ptBr from '../i18n/pt_BR.json'
+import messages_ru from '../i18n/ru.json'
+import messages_sv from '../i18n/sv.json'
 import messages_tr from '../i18n/tr.json'
-import messages_zhHant from '../i18n/zh_Hant.json'
 import messages_zhHans from '../i18n/zh_Hans.json'
+import messages_zhHant from '../i18n/zh_Hant.json'
 
-const supportedLanguages = ['de', 'fr', 'ja', 'nl', 'ru', 'es', 'oc', 'tr', 'zh-cn', 'zh-tw']
+const supportedLanguages = ['ca', 'de', 'el', 'en', 'es', 'fr', 'id', 'it', 'ja', 'nl', 'oc', 'pt-br', 'ru', 'sv', 'tr', 'zh-cn', 'zh-tw']
```
Comment (jespino) [inline webapp/src/i18n.tsx:22]: You are adding new languages here, but you are not adding them to the Languages menus (in GlobalHeaderMenu and in SidebarMenu)

**Review by jespino — CHANGES_REQUESTED** [review]
A couple of changes required.

**Review by wiggin77 — CHANGES_REQUESTED** [review]
@leandro-costa, will you be able to address the change request left by jespino?

**Comment by chenilim** [comment]
Trying to clean up stale PRs. @jespino, are there still changes needed? Or can they be made later? Thanks.

**Comment by jespino** [comment]
@chenilim I guess it can be done later, but I prefer to have everything done in the same PR.

**Pull request closed without merging**